lib/presentation/auth/widgets/enter_username.dart

```diff
@@ -115,17 +187,33 @@ class _EnterUserNameState extends State<EnterUserName> {
     );
   }
 
-  String? _validate(String? value) {
+  String? _validateFirstName(String? value) {
```

Comment:
Generalize `_validateFirstName`  and `_validateLastName` - They're basically the same functions.

Reply:
They have different String error messages, is there a way to identify TextFormField is it of Firstname or Lastname.

Reply:
To resolve added keys in Textfromfield, And then on the basis of keys display validation messages.